kubernetes/kfserving/kf_predictor_docker/config_kf.properties

```diff
@@ -0,0 +1,9 @@
+#Sample config.properties. In production config.properties at /mnt/models/config/config.properties will be used
+inference_address=http://0.0.0.0:8085
+management_address=http://0.0.0.0:8081
```

Comment:
NIT: Is there any reason for not exposing `metrics_address`?

Reply:
Added the metrics address. The kfserving wrapper does not implement metrics API as the 8082 port is not exposed in kfserver but the metrics API can be accessed inside the cluster.